packages/core/src/components/progress/progress.stories.ts

```diff
@@ -0,0 +1,71 @@
+import { htmlMatrix, Meta, omit, Story } from '../../../../../docs';
+import { Progress, ProgressProps } from './progress.story';
+
+const size = ['regular', 'large'] as const;
+
+export default {
+  title: 'Core/Progress',
+  component: Progress,
+  argTypes: {
+    children: {
+      description: 'Optional label.',
+    },
+    value: {
+      description: 'Current value',
+      table: {
+        defaultValue: { summary: '0' },
+      },
+    },
+    min: {
+      description: 'Optional minimum value',
+      table: {
+        defaultValue: { summary: '0' },
+      },
+    },
+    max: {
+      description: 'Optional maximum value',
+      table: {
+        defaultValue: { summary: '100' },
+      },
+    },
+    size: {
+      control: { type: 'radio', options: size },
+      table: {
+        type: { summary: size.toString() },
+        defaultValue: { summary: 'text' },
+      },
+    },
+    hideLabel: {
+      description: 'Hide label',
+      table: {
+        defaultValue: { summary: 'false' },
+      },
+    },
```

Comment:
No need to describe some obvious props:

```suggestion
    value: {
      table: {
        defaultValue: { summary: '0' },
      },
    },
    min: {
      table: {
        defaultValue: { summary: '0' },
      },
    },
    max: {
      table: {
        defaultValue: { summary: '100' },
      },
    },
    size: {
      control: { type: 'radio', options: size },
      table: {
        type: { summary: size.toString() },
        defaultValue: { summary: 'text' },
      },
    },
    hideLabel: {
      table: {
        defaultValue: { summary: 'false' },
      },
    },
```

Reply:
Done!